leakcanary-android-instrumentation/src/main/java/com/squareup/leakcanary/FailTestOnLeakRunListener.kt

```diff
@@ -41,7 +42,11 @@ open class FailTestOnLeakRunListener : RunListener() {
   private lateinit var bundle: Bundle
   private var skipLeakDetectionReason: String? = null
 
+  init {
+  }
```

Comment:
Is this needed?

Reply:
Nope, I pushed to review in github UI (so much better) and then only cleaned up